Gizmo/src/application/Main.java

```diff
@@ -102,6 +107,20 @@ public void handle(ActionEvent event) {
 		}
 	}
 	
+	class RedirectToSignup implements EventHandler<ActionEvent>{
+		public void handle(ActionEvent evento) {
+			Parent root = null;
+			try {
+				root = FXMLLoader.load(getClass().getResource("SignUp.fxml"));
```

Comment:
aise mat kar,

signUp button ka object bna in main, and then just use stage.setScene(signupObj.getScene());